docs/install-from-source.md

```diff
@@ -0,0 +1,8 @@
+# Install from Source
```

Comment:
IMO, this section it too short to have it's own page. I'd rather have it in the main README.

Reply:
Done.